### PR TITLE
Feature/no found matches

### PR DESCRIPTION
**Pull request by iRhysBradbury**

### CODE REVIEW

File: demo-ui/app/uk/gov/ons/addressIndex/demoui/views/MatchTypeHelper.scala [hunk 1]
```diff
@@ -4,7 +4,7 @@ object MatchTypeHelper {
 
   def matchType(addressId: String, ids: Seq[String], matchedFormattedAddress: String): String = {
     val matchedIds = ids.count(_ == addressId)
-    if(matchedIds == 1) {
+    if(matchedIds == 1 && !matchedFormattedAddress.isEmpty) {
```
Comment (Mironor) [inline demo-ui/app/uk/gov/ons/addressIndex/demoui/views/MatchTypeHelper.scala]: `!matchedFormattedAddress.isEmpty` is equivalent to `matchedFormattedAddress.nonEmpty`

Comment (iRhysBradbury) [inline demo-ui/app/uk/gov/ons/addressIndex/demoui/views/MatchTypeHelper.scala]: Thanks

**Comment by Mironor** [comment]
I wonder, if the problem is only with bulk search, why add "default" empty address on empty result to the repository instead of adding this `if` to the `bulkQuery` method ( here https://github.com/ONSdigital/address-index-api/blob/develop/server/app/uk/gov/ons/addressIndex/server/controllers/AddressController.scala#L213 )? 
This way we localize the change to the relevant functionality and avoid any regressions in other parts of the application.

**Comment by iRhysBradbury** [comment]
@Mironor agree - changing now

**Pull request merged**